package.json

```diff
@@ -31,5 +32,16 @@
     "stylus-loader": "^3.0.1",
     "webpack": "^2.2.1",
     "xstream": "^10.3.0"
+  },
+  "devDependencies": {
+    "husky": "^0.13.2",
+    "lint-staged": "^3.3.2",
+    "prettier": "^0.22.0"
+  },
+  "lint-staged": {
+    "*.js": [
+      "prettier --write",
+      "git add"
+    ]
```

Comment:
This workflow is really cool. Automatic writing of changes is kind of aggressive, but it's very expedient if you're on board with what writes are going to be made.